### ISSUE TITLE
After a while the IDS Connector is disconnected from the MDS Broker

### ISSUE THREAD

**Issue by edgardmarx** [Fraunhofer-IVI/Datenraum-Mobilitaet#4]
The issue is regarding connection instability related to the Production IDS Broker/Connector (https://broker.mobilitydataspace.io/).
Currently, I am using connector version 4.3.0.
After been connected for a while the Connector is not been shown anymore at the MDS Broker.
However, it is still connected, at least as shown in the log files.
To fix the issue, one must disconnect and connect again.

ps: the same issue was not verified in the MDS Test Broker

See attached screenshots.
![Screenshot 2021-07-13 at 14 47 42](https://user-images.githubusercontent.com/4638593/125454255-ab537d54-0636-4eca-ab68-fe4993ef660b.png)
![Screenshot 2021-07-13 at 14 47 52](https://user-images.githubusercontent.com/4638593/125454303-2ddf6fe5-a2b9-450e-8fdc-c6389cf125f9.png)

**Comment by sebplorenz** [comment]
You're right. This is an issue with the Broker version currently deployed on Demo stage. The latest version, that is currently deployed on Test stage and will be deployed to Demo stage soon, does not have this behavior.
Regards, Sebastian

**Comment by edgardmarx** [comment]
This is still happening, please reopen the issue.
Thanks

**Comment by sebplorenz** [comment]
Reopened until the new version is deployed on Demo.

**Comment by sebplorenz** [comment]
Btw. @edgardmarx Do you still have this issue with the Test or Demo Broker?

**Comment by edgardmarx** [comment]
Hi @sebplorenz,

thanks for the fast reply.
I am not sure whitch is the Demo or test Broker, but the issue was observed with the broker at https://broker.test.mobilitydataspace.io, which was updated few days ago at 20.8.2021 .
